### PR TITLE
Documenation by the uninitiated and handling private dependencies

### PR DESCRIPTION
**Pull request by jotelha**
Filling in documentation bits while deploying a testing instance without prior experience. Making non-public dependencies possible.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -125,7 +125,9 @@ Run the whole stack with
 
    docker compose up
 
-The stack automatically intitializes the database and creates and S3 bucket.
+The stack automatically initializes the database and creates an S3 bucket.
+
+You are now able to log in with via ORCID and upload data, but you will not have access to any analysis functionality.
```
Comment (mcrot) [inline README.rst:130]: Here I would add "so far", so it's clear, this can be added.

File: README.rst [hunk 2]
```diff
@@ -149,7 +151,7 @@ is linked to the group :code:`all` and add permissions for all commonly availabl
    You have to replace :code:`your_username` with the correct username.
    In order to find it, login with your ORCID
    and enter the "User Profile" page and take the last part of the URL.
-   Example: If the URL is :code:`https://localhost:8000/users/anna/`, then :code:`your_username` is :code:`anna`.
+   Example: If the URL is :code:`https://contact.engineering/users/anna/`, then :code:`your_username` is :code:`anna`.
```
Comment (mcrot) [inline README.rst:154]: No, the url with `localhost` is correct. The username given `contact.engineering` can differ and this is about the local development system.

File: README.rst [hunk 3]
```diff
@@ -159,7 +161,30 @@ is linked to the group :code:`all` and add permissions for all commonly availabl
...
+List all submodules in the :code:`.envs/.django` in a line
+
+.. code-block::
+
+    TOPOBANK_PLUGINS="topobank-statistics topobank-contact"
+
+separated by whitespace.
+
+When requirements in submodules change, update
+:code:`requirements/development.txt` by providing :code:`pip-compile`
```
Comment (mcrot) [inline README.rst:183]: Why prodiving `pip-compile`? Running `make` already calls `pip-compile`.

**Review by mcrot — CHANGES_REQUESTED** [review]
The old url was correct for the purpose of the readme, and I've 
propose some changes. Please see comments...

File: README.rst (hunk 3 above)
Comment (jotelha) [reply README.rst:184]: Yes, make calls it, but does not provide it. Won't run if it's not there.

File: README.rst (hunk 2 above)
Comment (jotelha) [reply README.rst:154]: That was something that reentered because of diverging branches

**Comment by jotelha** [comment]
@mcrot addressed your comments and renamed the pull request. Avoids embedding requirements file in container images.

**Review by jotelha — COMMENTED** [review]
Addressed all comments.

**Pull request merged**